pkg/manifests/nginx_test.go

```diff
@@ -22,6 +22,8 @@ var (
 				"service.beta.kubernetes.io/azure-load-balancer-internal": "true",
 			},
 		},
+		MinReplicas: 2,
```

Comment:
can you add a fixture test for when both scale values are nil? That's a possible case, right?

Reply:
that is not. the options struct used to generate manifest doesn't use a pointer.

the nil is transformed at a higher level where this is fully tested

Reply:
got it - and that's in the controller tests right?

Reply:
exactly